test/clt-tests/sharding/replication/data-manipulation.rec

```diff
@@ -0,0 +1,53 @@
+––– block: ./init –––
+––– input –––
+export INSTANCE=1
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=2
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export CLUSTER_NAME=c
+––– output –––
+––– block: ../base/flexible/create-cluster –––
+––– block: ../base/flexible/join-cluster-on-all-nodes –––
+––– input –––
+sleep 5; mysql -h0 -P1306 -e "create table c:testrt(id bigint, title text, content text, gid uint) shards=2 rf=1;"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "show tables;"
+––– output –––
++----------------+-------------+
+| Index          | Type        |
++----------------+-------------+
+| sharding_queue | rt          |
+| sharding_state | rt          |
+| sharding_table | rt          |
+| testrt         | distributed |
+| testrt_s%{NUMBER}      | rt          |
++----------------+-------------+
+––– input –––
+mysql -h0 -P1306 -e "INSERT INTO c:testrt (id, title, content, gid) values (1, 'Title 1', 'Content 1', 1), (2, 'Title 2', 'Content 2', 2);"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "INSERT INTO c:testrt (id, title, content, gid) values (3, 'Title 3', 'Content 3', 3);"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "INSERT INTO c:testrt (id, title, content, gid) values (4, 'Title 4', 'Content 4', 4);"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "SELECT * FROM c:testrt WHERE id IN (1, 2, 3, 4);"
```

Comment:
It's a good idea to add ORDER BY I think to make sure sorting

Reply:
Done